app/models/participant.rb

```diff
@@ -0,0 +1,5 @@
+class Participant < ApplicationRecord
+  belongs_to :user
+  belongs_to :stamp_rally
+  has_many :shop_participants, through: :stamp_card
```

Comment:
quick question - is stamp_card supposed to be PLURAL?

Reply:
I think so!!!